packages/client/src/graphql/auth/auth.graphql

```diff
@@ -0,0 +1,9 @@
+mutation loginEmail($projectId: String! $email: String!, $password: String!) {
```

Comment:
NIT: Is this file used? Looks like the mutation is replicated in the `auth.ts`

Reply:
It is the src of the graphql hooks. You write the graphql you want in any graphql file and then run`npm run introspection` to produced the corresponding ts files.